User/likravit/docs/whitepaper updates202004 (#587)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper.md

> You can turn off **Show me search and site suggestions using my typed characters** setting and change your default search engine from **edge://settings/search**.  While browsing InPrivate or in Guest mode, autosuggestions are turned off.  InPrivate will show suggestions from your local browsing such as browsing history or past searches, but no typed characters are sent to your default search engine.  Guest mode does not display any suggestions or send typed characters to your default search engine.

Reviewer: ```suggestion
You may turn off the **Show me search and site suggestions using my typed characters** setting and change your default search engine from `edge://settings/search`.  While browsing InPrivate or in Guest mode, autosuggestions are turned off.  InPrivate shows suggestions from your local browsing such as browsing history or past searches, but no typed characters are sent to your default search engine.  Guest mode does not display any suggestions or send typed characters to your default search engine.
```

> Data collected by other search providers follow their company's privacy policy.  

Reviewer: ```suggestion
Data collected by other search providers follow the privacy policy of the company.  
```

## Collections

> You can collect sites, text and images on the web and organize the content with Collections in Microsoft Edge.  All collections data is stored locally on the device and organized per Microsoft Edge profile.  If you have sync turned on for Collections, your collections created, including any notes or comments, will be available across all signed in and syncing versions of Microsoft Edge.  

Reviewer: ```suggestion
You may collect sites, text, and images on the web and organize the content with Collections in Microsoft Edge.  All collections data is stored locally on the device and organized per Microsoft Edge profile.  If you have sync turned on for Collections, your collections created, including any notes or comments, are available across all signed in and syncing versions of Microsoft Edge.  
```

> Every 24 hours Microsoft Edge downloads a list of supported sites for which special entity extraction templates exists.  The templates are specific to each website.  When you create a new item in your collection, Microsoft Edge will check to see if the site you are creating the new collection item from is on the list of supported sites.  If the site is on the list, Microsoft Edge will ping the entity extraction service for the specific sites template.  No user identifiers are associated with this request to the service.  This template attempts to identify the name, price, ratings, primary image, and other data about the item being collected.  If the site from which you are creating a new collection item is not on the supported list site, Microsoft Edge will not download a template.  The templates enable all collection items to be created locally on the device.  No data about the collection items are sent to the service to create the collection.

Reviewer: ```suggestion
Every 24 hours Microsoft Edge downloads a list of supported sites for which special entity extraction templates exists.  The templates are specific to each website.  When you create a new item in your collection, Microsoft Edge checks to see if the site you are creating the new collection item from is on the list of supported sites.  If the site is on the list, Microsoft Edge pings the entity extraction service for the specific sites template.  No user identifiers are associated with this request to the service.  This template attempts to identify the name, price, ratings, primary image, and other data about the item being collected.  If the site from which you are creating a new collection item is not on the supported list site, Microsoft Edge does not download a template.  The templates enable all collection items to be created locally on the device.  No data about the collection items are sent to the service to create the collection.
```

> The diagnostic data is stored on Microsoft servers for up to 18 months.  On Windows 10 you can delete this diagnostic data by going to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback** and selecting **Delete** under **Delete diagnostic data**.  The Delete diagnostic data functionality is only supported on Windows 10, April 2018 update and newer.  For more information see [Diagnostics, feedback, and privacy in Windows 10](https://support.microsoft.com/help/4468236/diagnostics-feedback-and-privacy-in-windows-10-microsoft-privacy).  

Reviewer: ```suggestion
The diagnostic data is stored on Microsoft servers for up to 18 months.  On Windows 10, you may delete this diagnostic data by going to **Start** > **Settings** > **Privacy** > **Diagnostics \& feedback** and selecting **Delete** under **Delete diagnostic data**.  The Delete diagnostic data functionality is only supported on Windows 10, April 2018 update and newer.  For more information, see [Diagnostics, feedback, and privacy in Windows 10](https://support.microsoft.com/help/4468236/diagnostics-feedback-and-privacy-in-windows-10-microsoft-privacy).  
```

> For Windows 10 Stable and Beta channels of Microsoft Edge, this setting is determined by your Windows diagnostic data setting.  This is reflected in your Microsoft Edge Privacy and services settings.  You can change this Windows settings by going to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**.  On all other channels and platforms, you can control this diagnostic data collection by going to **edge://settings/privacy** and turning on or off **Improve Microsoft products by sending crash reports and data about how you use the browser**.  This setting is the same for all profiles associated with the installation of Microsoft Edge on your device.  This setting is not synced across devices.  The setting applies to InPrivate browsing and Guest mode.  If your device is managed with group policies set by your organization, this will be reflected in **edge://settings/privacy**.  

Reviewer: ```suggestion
For Windows 10 Stable and Beta channels of Microsoft Edge, this setting is determined by your Windows diagnostic data setting.  This is reflected in your Microsoft Edge Privacy and services settings.  You may change this Windows settings by going to **Start** > **Settings** > **Privacy** > **Diagnostics \& feedback**.  On all other channels and platforms, you may control this diagnostic data collection by going to `edge://settings/privacy` and turning on or off the **Improve Microsoft products by sending crash reports and data about how you use the browser** setting.  This setting is the same for all profiles associated with the installation of Microsoft Edge on your device.  This setting is not synced across devices.  The setting applies to InPrivate browsing and Guest mode.  If your device is managed with group policies set by your organization, this is reflected in `edge://settings/privacy`.  
```

> When you turn on **Send "Do Not Track" requests** in **edge://settings/privacy**, Microsoft Edge will send a DNT:1 HTTP header with your outgoing HTTP, HTTPS and SPDY browsing traffic requests to websites you visit to request that they not use trackers.  However, turning on **Send "Do Not Track" requests** doesn't guarantee that websites won't track you.  Some sites might honor this request by showing you ads not based on any previous browsing.  Microsoft Edge does not have control if this request will be honored or not.  To help prevent websites from tracking you, change your **Tracking prevention** settings in **edge://settings/privacy** to **Balanced** or **Strict**.  

Reviewer: ```suggestion
When you turn on the **Send "Do Not Track" requests** setting in `edge://settings/privacy`, Microsoft Edge sends a DNT:1 HTTP header with your outgoing HTTP, HTTPS, and SPDY browsing traffic requests to websites you visit to request that the sites not use trackers.  However, turning on the **Send "Do Not Track" requests** setting doesn't guarantee that websites are not tracking you.  Some sites may honor the request by showing you ads not based on any previous browsing.  Microsoft Edge does not have control if this request is honored or not.  To help prevent websites from tracking you, change your **Tracking prevention** settings in `edge://settings/privacy` to **Balanced** or **Strict**.  
```

> When you use guest mode, Microsoft Edge won't send "Do Not Track" requests.  When you use InPrivate browsing, Microsoft Edge will only send "Do Not Track" requests if **Send "Do Not Track" requests** is turned on for the profile you're using.  

Reviewer: ```suggestion
When you use guest mode, Microsoft Edge does not send "Do Not Track" requests.  When you use InPrivate browsing, Microsoft Edge sends only  "Do Not Track" requests if the **Send "Do Not Track" requests** setting is turned on for the profile you're using.  
```

## Feedback

> We're always listening to customers and value your feedback.  To provide feedback in Microsoft Edge, select the **Settings and more** > **Help and feedback** > **Send feedback**.  For Progressive Web Apps (PWA), select the Settings and more ...  -> Send feedback to Microsoft.  You must provide details about the feedback, but all other information is optional.  If an email is detected from your Microsoft Edge profile it is prepopulated along with the current URL of the site you are on and relevant diagnostic data.  The diagnostic data can include data about the features you've turned on and your usage of the browser.  A screenshot, file from your device, and recording of your browser can also be optionally included.  The optional additional recording you provide may include personally identifiable information.  This data is only used for diagnostic and product improvement purposes.  

Reviewer: ```suggestion
Microsoft is always listening to customers and values your feedback.  To provide feedback in Microsoft Edge, select the **Settings and more** > **Help and feedback** > **Send feedback**.  For Progressive Web Apps (PWA), select the **Settings and more** > **Send feedback to Microsoft**.  You must provide details about the feedback, but all other information is optional.  If an email is detected from your Microsoft Edge profile, it is prepopulated along with the current URL of the site you are on and relevant diagnostic data.  The diagnostic data may include data about the features that you turned on and your usage of the browser.  A screenshot, file from your device, and recording of your browser may also be optionally included.  The optional additional recording you provide may include personally identifiable information.  This data is only used for diagnostic and product improvement purposes.  
```

> User feedback is securely sent to Microsoft via HTTPS and stored on secure Microsoft servers.  If you include your email address and  **Improve Microsoft products by sending crash reports and data about how you use the browser** is turned on in your Microsoft Edge privacy settings, an identifier unique to your browser installation on your device will be associated with your feedback.  All diagnostic data including diagnostic logs, recordings, and attachments are stored for up to 30 days.  The remaining feedback data including an optional screenshot is stored for up to 15 months.  You can make a [request](https://www.microsoft.com/concern/privacy) to delete your feedback if you provided an email with your feedback item.  

Reviewer: ```suggestion
User feedback is securely sent to Microsoft via HTTPS and stored on secure Microsoft servers.  If you include your email address and  the **Improve Microsoft products by sending crash reports and data about how you use the browser** setting is turned on in your Microsoft Edge privacy settings, an identifier unique to your browser installation on your device is associated with your feedback.  All diagnostic data including diagnostic logs, recordings, and attachments are stored for up to 30 days.  The remaining feedback data including an optional screenshot is stored for up to 15 months.  You may make a [request](https://www.microsoft.com/concern/privacy) to delete your feedback if you provided an email address with your feedback item.  
```

> InPrivate browsing will use the location permission setting of the profile from which the InPrivate session was launched.  Guest mode will always ask you for permission before granting the site your location.

Reviewer: ```suggestion
InPrivate browsing uses the location permission setting of the profile from which the InPrivate session was launched.  Guest mode always asks you for permission before granting the site your location.
```

## Import browser data

> Microsoft Edge offers an interactive and seamless experience when you launch the browser for the first time.  As part of this experience, you have the option to import your browser data to Microsoft Edge from another browser.  During the import experience you can either keep your imported data or delete it and start fresh.  This data includes your favorites, browsing history, autofill data, extensions, settings, and other browsing data.  Your browsing data from older versions of Microsoft Edge is automatically included when you update to the new Microsoft Edge.  With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox and Internet Explorer and is determined based on your most used browser defined by your OS.  Importing your data is all completed locally on your device, stored locally and is not sent to Microsoft unless you agree to syncing your browsing data.  

Reviewer: ```suggestion
Microsoft Edge offers an interactive and seamless experience when you launch the browser for the first time.  As part of this experience, you have the option to import your browser data to Microsoft Edge from another browser.  During the import experience you may either keep your imported data or delete it and start fresh.  This data includes your favorites, browsing history, autofill data, extensions, settings, and other browsing data.  Your browsing data from older versions of Microsoft Edge is automatically included when you update to the new Microsoft Edge.  With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, and Internet Explorer and is determined based on your most used browser defined by your OS.  Importing your data is all completed locally on your device, stored locally, and is not sent to Microsoft unless you agree to syncing your browsing data.  
```

> ![Import](./media/WhitepaperMedia/migration.png)

Reviewer: ```suggestion
![Import](./media/whitepapermedia/migration.png)
```

> When importing your extensions from a different browser such as Google Chrome, we will import a local copy and ask for permission before it is turned on.  The permissions for some of the extensions may have changed.  To review the extension permissions, go to **edge://extensions**.  

Reviewer: ```suggestion
When importing your extensions from a different browser such as Google Chrome, The Microsoft Edge imports a local copy and asks for permission before it is turned on.  The permissions for some of the extensions may have changed.  To review the extension permissions, go to `edge://extensions`.  
```

> You can choose to import data from another browser at any time by going to **edge://settings/importData**.

Reviewer: ```suggestion
You may choose to import data from another browser at any time by going to `edge://settings/importData`.
```

## Install and update

> You can download and install Microsoft Edge on platforms such as Windows and macOS.  Microsoft Edge uses the updater service to keep your version of Microsoft Edge up to date and secure.  

Reviewer: ```suggestion
You may download and install Microsoft Edge on platforms such as Windows and macOS.  Microsoft Edge uses the updater service to keep your version of Microsoft Edge up to date and secure.  
```

> If your administrator turned on Enterprise Site Discovery, browsing history data is collected to help administrators review the sites that users visit periodically and ensure that any system upgrades continue to support those sites.  [Learn more](https://go.microsoft.com/fwlink/?linkid=2114123) 

Reviewer: ```suggestion
If your administrator turned on Enterprise Site Discovery, browsing history data is collected to help administrators review the sites that users visit periodically and ensure that any system upgrades continue to support those sites.  To learn more, see [Collect Data Using Enterprise Site Discovery](/internet-explorer/ie11-deploy-guide/collect-data-using-enterprise-site-discovery).   
```

> Internet Explorer browsing is stored locally in Microsoft Edge and Internet Explorer.  This data can be cleared from Clear browsing data and Clear browsing data for Internet Explorer in **edge://settings/privacy**.  

Reviewer: ```suggestion
Internet Explorer browsing is stored locally in Microsoft Edge and Internet Explorer.  This data may be cleared from Clear browsing data and Clear browsing data for Internet Explorer in `edge://settings/privacy`.  
```

### Office 365

> If you're signed in to Microsoft Edge with a work or school account, your organization may turn on Office 365 as an option for page content on the new tab page.  This feature is currently available only for commercial customers, and is governed under the [Microsoft Online Services Terms (OST)](https://www.microsoft.com/licensing/product-licensing/products).  [Learn more about privacy for Office 365](https://docs.microsoft.com/deployoffice/privacy/overview-privacy-controls).

Reviewer: ```suggestion
If you're signed in to Microsoft Edge with a work or school account, your organization may turn on Office 365 as an option for page content on the new tab page.  This feature is currently available only for commercial customers, and is governed under the [Microsoft Online Services Terms (OST)](https://www.microsoft.com/licensing/product-licensing/products).  To more about privacy for Office 365, see [Overview Of Privacy Controls For Office 365 ProPlus](/deployoffice/privacy/overview-privacy-controls).  
```

> Microsoft Edge supports the PaymentRequest API by letting you pay for purchases with payment info you've saved using autofill.  PaymentRequest API allows the merchant to request the following information: credit card number, credit card expiration, full name, billing address, email address, phone number, and shipping address.  The API tells the merchant that the user has credit card info saved but doesn't share any info with the merchant until you agree.  You can turn off this feature in **edge://settings/privacy**.  

Reviewer: ```suggestion
Microsoft Edge supports the PaymentRequest API by letting you pay for purchases with payment info that you saved using autofill.  PaymentRequest API allows the merchant to request the following information: credit card number, credit card expiration, full name, billing address, email address, phone number, and shipping address.  The API tells the merchant that the user has credit card info saved but doesn't share any info with the merchant until you agree.  You can turn off this feature in `edge://settings/privacy`.  
```

> If you've saved payment info to your Microsoft account, it's also available for autofill in the browser.  Payment info stored in your Microsoft account syncs across devices.  If you've made any Xbox or Microsoft Store purchases in the past, you might already have payment info saved to your Microsoft account.  During payment autofill, a card from your Microsoft account is masked and is only fully revealed after two-factor authentication.  This provides additional security while retrieving your payment info.  

Reviewer: ```suggestion
If you saved payment info to your Microsoft account, it is also available for autofill in the browser.  Payment info stored in your Microsoft account syncs across devices.  If you made any Xbox or Microsoft Store purchases in the past, you may already have payment info saved to your Microsoft account.  During payment autofill, a card from your Microsoft account is masked and is only fully revealed after two-factor authentication.  This provides additional security while retrieving your payment info.  
```

> Microsoft Edge lets you print webpages, PDF files, or other content using a variety of devices and applications.  When you print to a printer, application, or PDF, Microsoft Edge sends the commands and file info to your device's operating system.  This info is not sent to Microsoft, and all data sent to your device's operating system for printing is deleted immediately after printing is completed or canceled.  To change your printing destination, go to **edge://settings/printing**.  

Reviewer: ```suggestion
Microsoft Edge lets you print webpages, PDF files, or other content using a variety of devices and applications.  When you print to a printer, application, or PDF, Microsoft Edge sends the commands and file info to the operating system of your device.  This info is not sent to Microsoft, and all data sent to the operating system of your device for printing is deleted immediately after printing is completed or canceled.  To change your printing destination, go to `edge://settings/printing`.  
```

> You can also print webpages and files to PDF format using Microsoft Print to PDF, which does not send any data about the file back to Microsoft.  Any annotations made to the PDF file are saved locally to the file itself.  

Reviewer: ```suggestion
You may also print webpages and files to PDF format using Microsoft Print to PDF, which does not send any data about the file back to Microsoft.  Any annotations made to the PDF file are only saved locally to the file.  
```

> When you start Read aloud, Microsoft Edge will use the [Web Speech API](https://wicg.github.io/speech-api/).  Depending on the voice you select, the content of the page will be converted from text to speech using either a platform-supplied, client-side library (i.e.  one specific to your operating system) or a server-side library powered by Microsoft Cognitive Services.  If your content is converted to speech using a client-side library, no information will be sent to Microsoft servers.  If your content is converted to speech using Microsoft Cognitive Services (as indicated by the word "Online" in any of the voice names), the text, along with a randomly generated token, is sent to Microsoft.  Once conversion is complete, the service returns the spoken text in an audio file to your device.  All data is encrypted while being transferred from your device to Microsoft and vice versa.  The text that is sent to Microsoft and the audio file that is generated are both deleted immediately after conversion occurs; no other data about your web content is stored for any period of time.

Reviewer: ```suggestion
When you start Read aloud, Microsoft Edge uses the [Web Speech API](https://wicg.github.io/speech-api/).  Depending on the voice you select, the content of the page will be converted from text to speech using either a platform-supplied, client-side library \(for example, one specific to your operating system\) or a server-side library powered by Microsoft Cognitive Services.  If your content is converted to speech using a client-side library, no information is sent to Microsoft servers.  If your content is converted to speech using Microsoft Cognitive Services \(as indicated by the word "Online" in any of the voice names\), the text, along with a randomly generated token, is sent to Microsoft.  Once conversion is complete, the service returns the spoken text in an audio file to your device.  All data is encrypted while being transferred from your device to Microsoft and vice versa.  The text that is sent to Microsoft and the audio file that is generated are both deleted immediately after conversion occurs; no other data about your web content is stored for any period of time.
```

## Releasing new functionality 

> To improve Microsoft Edge, we're always learning from our users.  As part of this learning, some users may experience new functionality before it's made available to everyone.  To enable new functionality for randomly selected users, Microsoft Edge regularly sends required info about your operating system, channel, version, country or region, and other device configuration data to the Microsoft Edge configuration service.  This data is sent with a resettable identifier unique to your browser.  Data is transmitted to the service over HTTPS.  This data is used for receiving updates to enable new functionality, to keep Microsoft Edge up to date and performing properly, and to improve our products and services.  Additional controls and configurations are available for organizations.  [Learn more](https://go.microsoft.com/fwlink/?linkid=2112125) 

Reviewer: ```suggestion
To improve Microsoft Edge, the Microsoft Edge team is always learning from our users.  As part of this learning, some users may experience new functionality before it is made available to everyone.  To enable new functionality for randomly selected users, Microsoft Edge regularly sends required info about your operating system, channel, version, country or region, and other device configuration data to the Microsoft Edge configuration service.  This data is sent with a resettable identifier unique to your browser.  Data is transmitted to the service over HTTPS.  This data is used for receiving updates to enable new functionality, to keep Microsoft Edge up to date and performing properly, and to improve our products and services.  Additional controls and configurations are available for organizations.  To learn more about controls and configurations, see [Microsoft Edge Configurations And Experimentation](/deployedge/edge-configuration-and-experiments) 
```

> As a user, you can't turn off these browser updates, but you can control whether your product usage data is sent to Microsoft by going to **edge://settings/privacy** and changing the settings under **Help improve Microsoft Edge**.  

Reviewer: ```suggestion
As a user, you are not able to turn off these browser updates, but you may control whether your product usage data is sent to Microsoft by going to `edge://settings/privacy` and changing the settings under **Help improve Microsoft Edge**.  
```

> To understand how new functionality affects Microsoft Edge and Microsoft services, Microsoft Edge sends a resettable identifier unique to your browser and a functionality tag that encodes what new functionality was enabled to Microsoft Edge and Microsoft services.  This helps us build experiences and the best browser for everyone.  The functionality tag is not unique to your installation of Microsoft Edge and is shared across all Microsoft Edge installations that share the same set of functionality being enabled.  Microsoft Edge sends this information in a HTTPS header to only Microsoft services.  The browser does not send this header when you browse InPrivate or in Guest mode.  You can prevent this data from being sent by going to **edge://settings/privacy** and turning off **Improve Microsoft products by sending crash reports and data about how you use the browser**.  [Learn how to reset the identifier unique to your browser](#Diagnostic-data-about-how-you-use-the-browser).  

Reviewer: ```suggestion
To understand how new functionality affects Microsoft Edge and Microsoft services, Microsoft Edge sends a resettable identifier unique to your browser and a functionality tag that encodes what new functionality was enabled to Microsoft Edge and Microsoft services.  This helps us build experiences and the best browser for everyone.  The functionality tag is not unique to your installation of Microsoft Edge and is shared across all Microsoft Edge installations that share the same set of functionality being enabled.  Microsoft Edge sends this information in a HTTPS header to only Microsoft services.  The browser does not send this header when you browse InPrivate or in Guest mode.  You may prevent this data from being sent by going to `edge://settings/privacy` and turning off the **Improve Microsoft products by sending crash reports and data about how you use the browser** setting.  To learn how to reset the identifier unique to your browser, see [Diagnostic data about how you use the browser](#diagnostic-data-about-how-you-use-the-browser).  
```

## Resolve navigation errors

> If Microsoft Edge detects SSL connection timeouts, certificate errors,or other network issues that might be caused by a captive portal (for example, a Wi-Fi network at a hotel or airport), Microsoft Edge will make a request to `http://edge.microsoft.com/captiveportal/generate_204` and check the response code.  If the request is redirected to another URL, Microsoft Edge will open that URL in a new tab, assuming that it's a sign-in page.  Requests to the captive portal detection page is a stateless service, and requests aren't logged and cookies aren't sent or saved.  On Windows platforms, Microsoft Edge will use a Windows captive portal service.  Otherwise, the Microsoft Edge captive portal service is used.  To turn this service off, go to **edge://settings/privacy** and turn off **Use a web service to help resolve navigation errors**.

Reviewer: ```suggestion
If Microsoft Edge detects SSL connection timeouts, certificate errors,or other network issues that may be caused by a captive portal \(for example, a Wi-Fi network at a hotel or airport\), Microsoft Edge makes a request to `http://edge.microsoft.com/captiveportal/generate_204` and checks the response code.  If the request is redirected to another URL, Microsoft Edge opens that URL in a new tab, assuming that it is a sign-in page.  Requests to the captive portal detection page is a stateless service, and requests aren't logged and cookies aren't sent or saved.  On Windows platforms, Microsoft Edge uses a Windows captive portal service.  Otherwise, the Microsoft Edge captive portal service is used.  To turn this service off, go to `edge://settings/privacy` and turn off the **Use a web service to help resolve navigation errors** setting.
```

> Your identity in Microsoft Edge on MacOS is shared between Microsoft apps.  This allows you to sign in to a Microsoft app without having to separately enter your credentials if you're signed in to another Microsoft app on the device.  On MacOS, you won't be automatically signed in to Microsoft Edge based on your authentication state in another Microsoft app.  When you try to sign in to Microsoft Edge, we'll offer to use the credentials from another Microsoft app to sign you in to Microsoft Edge seamlessly.  Similarly, when you are signed in with an account to Microsoft Edge, if you try to sign into other Microsoft apps, your Microsoft Edge credentials can be used to help you sign in to those other Microsoft apps on the device without requiring you to enter your credentials again.  

Reviewer: ```suggestion
Your identity in Microsoft Edge on macOS is shared between Microsoft apps.  This allows you to sign into a Microsoft app without having to separately enter your credentials if you're signed in to another Microsoft app on the device.  On macOS, you are not automatically signed into Microsoft Edge based on your authentication state in another Microsoft app.  When you try to sign into Microsoft Edge, Microsoft Edge offers to use the credentials from another Microsoft app to sign you into Microsoft Edge seamlessly.  Similarly, when you are signed into an account to Microsoft Edge, if you try to sign into another Microsoft app, your Microsoft Edge credentials may be used to help you sign into the other Microsoft app on the device without requiring you to enter your credentials again.  
```

> You can't sign in to Microsoft Edge when using Guest mode or InPrivate browsing.  

Reviewer: ```suggestion
You are not able to sign into Microsoft Edge when using Guest mode or InPrivate browsing.  
```

The SmartScreen service stores data about the reputation checks and builds a database of known malicious URLs and files.  This data is stored on secure Microsoft servers and is only used for Microsoft security services.  This data will never be used to identify or target you in any way.  Clearing your browsing cache clears all locally stored SmartScreen URL data.  Clearing your download history will remove any locally stored SmartScreen data about file downloads.

> SmartScreen is turned on by default for Microsoft Edge.  To turn off SmartScreen, go to **edge://settings/privacy** > **Services** > **Microsoft Defender SmartScreen**.  This setting is the same for all profiles associated with the installation of Microsoft Edge on your device.  This setting is not synced across devices.  The setting applies to InPrivate browsing and Guest mode.  If your device is managed with group policies set by your organization, this will be reflected in **edge://settings/privacy**.  [Learn more about SmartScreen](https://support.microsoft.com/help/17443).  

Reviewer: ```suggestion
SmartScreen is turned on by default for Microsoft Edge.  To turn off SmartScreen, go to `edge://settings/privacy` under **Services** > **Microsoft Defender SmartScreen**.  This setting is the same for all profiles associated with the installation of Microsoft Edge on your device.  This setting is not synced across devices.  The setting applies to InPrivate browsing and Guest mode.  If your device is managed with group policies set by your organization, this is reflected in `edge://settings/privacy`.  To learn more about SmartScreen, see [SmartScreen: FAQ](https://support.microsoft.com/help/17443).  
```

## Sync

> Signing in to Microsoft Edge with a Microsoft account allows you to sync your browsing data across all your signed-in versions of Microsoft Edge.  You can sync your favorites, settings, form fill data (addresses and more), and passwords.  You must allow permission to turn on sync in Microsoft Edge, and each synced data type can be turned on or off individually.  Favorites includes any tabs you previously set aside in the legacy version of Microsoft Edge, and will sync along with your favorites.  Deleting or modifications to favorites, and other data types from one signed-in version of Microsoft Edge will sync to all other signed-in versions of Microsoft Edge where sync is turned on.  Sync configurations can be managed from **edge://settings/profiles/sync**.  

Reviewer: ```suggestion
Signing into Microsoft Edge with a Microsoft account allows you to sync your browsing data across all your signed-in versions of Microsoft Edge.  You may sync your favorites, settings, form fill data \(addresses and more\), and passwords.  You must allow permission to turn on sync in Microsoft Edge, and each synced data type can be turned on or off individually.  Favorites includes any tabs you previously set aside in the legacy version of Microsoft Edge, and syncs along with your favorites.  Deleting or modifications to favorites, and other data types from one signed-in version of Microsoft Edge syncs to all other signed-in versions of Microsoft Edge where sync is turned on.  Sync configurations may be managed from `edge://settings/profiles/sync`.  
```

> ![Sync](./media/WhitepaperMedia/Sync.png)

Reviewer: ```suggestion
![Sync](./media/whitepapermedia/sync.png)
```

> For sync to function, there is additional device connectivity and config data such as name of your device, make and model, necessary to provide the sync experience.  This data can be deleted from the [Microsoft device dashboard](https://account.microsoft.com/devices/).  You can manage synced favorites from **edge://favorites**.  All other data types can be managed from your Microsoft Edge profile settings at **edge://settings/profiles**.  

Reviewer: ```suggestion
For sync to function, there is additional device connectivity and config data such as name of your device, make and model, necessary to provide the sync experience.  This data may be deleted from the [Microsoft device dashboard](https://account.microsoft.com/devices/).  You may manage synced favorites from `edge://favorites`.  All other data types may be managed from your Microsoft Edge profile settings at `edge://settings/profiles`.  
```

Microsoft Edge allows you to install web apps made by website developers and pin your favorite sites.  

> When you pin a site, it will be added to your taskbar or dock.  This data is stored locally on your device.  For some sites, info about whether the site has been pinned is shared with the site, so the site knows not to prompt to be pinned.  You can manage your pinned sites from the taskbar or dock.  Pinned sites open in Microsoft Edge windows and use the same site permissions and diagnostic data about how you use the browser and sites you visit as that version of Microsoft Edge.  

Reviewer: ```suggestion
When you pin a site, it will be added to your taskbar or dock.  This data is stored locally on your device.  For some sites, info about whether the site has been pinned is shared with the site, so the site knows not to prompt to be pinned.  You may manage your pinned sites from the taskbar or dock.  Pinned sites open in Microsoft Edge windows and use the same site permissions and diagnostic data about how you use the browser and sites you visit as that version of Microsoft Edge.  
```

## WebView

> Microsoft Edge WebView allows app developers to host web content in their native applications on Windows 10 and Windows 7.  The applications hosting the Microsoft Edge WebView may send diagnostic data about how you use the Microsoft Edge web platform and sites you visit in the application to Microsoft.  This diagnostic data collection is determined by your **Help improve Microsoft Edge** settings in Microsoft Edge Privacy and services settings.  To turn off this diagnostic data collection for Windows 10 Microsoft Edge Stable and Beta channels, go to Windows diagnostic data settings.  To turn off this diagnostic data collection for all other channels, go to **edge://settings/privacy** in a normal browsing session and turn off both **Improve Microsoft products by sending crash reports and data about how you use the browser** and **Make searches and Microsoft products better by sending info about websites you visit in Microsoft Edge.**  The applications hosting the Microsoft Edge WebView may collect other data and have their own data collection management and privacy policies.  

Reviewer: ```suggestion
Microsoft Edge WebView allows app developers to host web content in their native applications on Windows 10 and Windows 7.  The applications hosting the Microsoft Edge WebView may send diagnostic data about how you use the Microsoft Edge web platform and sites you visit in the application to Microsoft.  This diagnostic data collection is determined by your **Help improve Microsoft Edge** settings in Microsoft Edge Privacy and services settings.  To turn off this diagnostic data collection for Windows 10 Microsoft Edge Stable and Beta channels, go to Windows diagnostic data settings.  To turn off this diagnostic data collection for all other channels, go to `edge://settings/privacy` in a normal browsing session and turn off both **Improve Microsoft products by sending crash reports and data about how you use the browser** and **Make searches and Microsoft products better by sending info about websites you visit in Microsoft Edge** settings.  The applications hosting the Microsoft Edge WebView may collect other data and have individual data collection management and privacy policies.  
```
